packages/run-protocol/src/vaultFactory/addAssetToVault.js

```diff
@@ -124,36 +146,52 @@ export const registerScaledPriceAuthority = async ({
   );
   await E(priceAuthorityAdmin).registerPriceAuthority(
     E(publicFacet).getPriceAuthority(),
-    ibcAtomBrand,
+    interchainBrand,
     runBrand,
     true, // force
   );
 };
 
 /** @typedef {import('../econ-behaviors.js').EconomyBootstrapPowers} EconomyBootstrapPowers */
 
-/** @param {EconomyBootstrapPowers} powers */
-export const addAssetToVault = async ({
-  consume: { vaultFactoryCreator, reserveCreatorFacet, agoricNamesAdmin, zoe },
-  brand: {
-    consume: { RUN: runP },
+/**
+ * @param {EconomyBootstrapPowers} powers
+ * @param {object} config
+ * @param {object} [config.options]
+ * @param {string} [config.options.interchainKeyword]
+ * @param {string} [config.options.interchainOracle]
```

Comment:
add some prose docs for these?

Reply:
I created a separate type that should make this self-documenting.